src/components/security_manager/src/crypto_manager_impl.cc

```diff
@@ -345,8 +345,11 @@ bool CryptoManagerImpl::IsCertificateUpdateRequired(
   const double seconds = difftime(certificates_time, system_time);
 
   LOG4CXX_DEBUG(
-      logger_, "Certificate UTC time: " << asctime(gmtime(&certificates_time)));
+      logger_,
+      // cppcheck-suppress asctimeCalled
+      "Certificate UTC time: " << asctime(gmtime(&certificates_time)));
 
+  // cppcheck-suppress asctimeCalled
   LOG4CXX_DEBUG(logger_, "Host UTC time: " << asctime(gmtime(&system_time)));
```

Comment:
@Ypostolov For these three cases with `cppcheck-suppress asctimeCalled` you should include `<ctime>` library and use `strftime` function instead of `asctime`. You could find more info at http://www.cplusplus.com/reference/ctime/strftime/